### DISCUSSION TITLE
Is Pybricks allowed in FLL competitions?

### DISCUSSION THREAD

**Discussion by sarpedondev** [pybricks/support#1394] [Q&A] [answered]
Hi there,
My FLL Team and I were trying out pybricks on one of our Spike Primes today. It worked great; thanks for making such an amazing piece of software! However, one of our team members was not happy when he heard that we "modded" our hub. He claimed that a few years ago, some team in the international round in the USA got a 5-year ban for using a modded ev3 running pybricks. Additionally, he said that because we flashed the pybricks firmware, some electronic warranty piece on the board probably burned, and now the warranty is  void. I would love to know if any of you have experience with any of those topics.
Thanks, Tom

**Comment by laurensvalk** [comment, marked as answer]
Thanks for your message! Glad to hear Pybricks is working well.

Pybricks for EV3 was developed together with LEGO Education. The download is even hosted on the [official LEGO site](https://education.lego.com/en-us/product-resources/mindstorms-ev3/teacher-resources/python-for-ev3/), so it's easy enough to dispel that rumor :smile:

Maybe they were referring to some other program. A long time ago, alternative softwares were not allowed.

Alternative languages are allowed these days. (If you're reading this in the future - always check the latest rules).

As far as installation on SPIKE goes, we use the exact same technique as LEGO Education has recently used in some of their hub updates. Nothing changes to the hub, and you can go back to the original any time.

**Reply by sarpedondev** [reply]
Thank you very much for your answer!

**Comment by afarago** [comment]
Just to add the right pointer to the zillions pieces of info around FLL.
https://firstinspiresst01.blob.core.windows.net/first-in-show-masterpiece/fll-challenge/fll-challenge-masterpiece-rgr-en.pdf

"Teams can use any software or programming language. Robots must be autonomous during the match.
No remote controllers of any type are allowed."

Hope that helps you and your team to participate with confidence!
![image](https://github.com/pybricks/support/assets/4489389/efa7c03a-7c39-400d-8f76-1f961de1956c)

**Reply by laurensvalk** [reply]
Thanks for adding this! Maybe we should rename this issue to avoid starting yet another urban legend among teams :smile:

**Comment by sarpedondev** [comment]
> Thanks for adding this! Maybe we should rename this issue to avoid starting yet another urban legend among teams 😄

What do you suppose I should change it to?

**Reply by dlech** [reply]
"Is Pybricks allowed in FLL competitions?"
